### ISSUE TITLE
F.normalize and torch.linalg.norm

### ISSUE THREAD

**Issue by Seungwoo0326** [pytorch/pytorch#49152]
## To Reproduce

When you use torch.nn.functional.normalize, you will get
UserWarning: torch.norm is deprecated and may be removed in a future PyTorch release. Use torch.linalg.norm instead.
  "torch.norm is deprecated and may be removed in a future PyTorch release. "
this error.

## Expected behavior

I think you have to correct torch.nn.functional.normalize also, not to use torch.norm.
You should use torch.linalg.norm for this function, too.

## Environment

 - PyTorch Version 1.7
 - OS: Linux

**Comment by mruberry** [comment]
Thanks for reporting this issue, @Seungwoo0326! We'll get it fixed.

**Comment by mruberry** [comment]
Good news: F.normalize no longer throws a deprecation warning.
